build-utils/language-java/src/test/resources/integration-test/pom.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
same as other comment about test-pom.xml: is there a way to keep them easy in sync